software/bknd.yml

```
@@ -0,0 +1,15 @@
+name: "bknd"
+website_url: "https://bknd.io"
+source_code_url: "https://github.com/bknd-io/bknd"
+description: "Lightweight Firebase alternative built to run anywhere."
```

Description should describe what the project does and also matches the alternative formatting found here: https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines

```suggestion
description: "Simply app development by providing a fully functional backend for database management, authentication, media and workflows (alternative to Firebase)."
```